src/LiveDevelopment/Documents/HTMLDocument.js

```diff
@@ -63,7 +63,7 @@ define(function HTMLDocumentModule(require, exports, module) {
 
         this.editor = editor;
         this._instrumentationEnabled = false;
-
+        this._debug = false;
```

Comment:
Discussed on Mattermost, but I'll repeat here for searchability.  When we change things in the core Brackets code in ways that are only needed for Bramble, we've tended to add a `XXXBramble` comment, so as to make it clear why we're doing this at all.  It's not critical, but it's helpful when we do big upstream merges later.